app/src/main/java/com/yacgroup/yacguide/DescriptionActivity.kt

```diff
@@ -74,50 +74,45 @@ class DescriptionActivity : TableActivity() {
                         getString(R.string.route_quality),
                         RouteComment.QUALITY_MAP[comment.qualityId].orEmpty(),
                         WidgetUtils.textFontSizeDp,
-                        View.OnClickListener { },
+                        { },
```

Comment:
Maybe you should define this as default argument to avoid repetitive code.

Reply:
Done. Merging now...